config/jobs/kubernetes/sig-auth/sig-auth-encryption-at-rest.yaml

```diff
@@ -0,0 +1,50 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-kind-kms
+    decorate: true
+    decoration_config:
+      timeout: 150m
+    always_run: false
+    optional: true # TODO (aramase): make this required and add run_if_changed once the job is passing
+    path_alias: k8s.io/kubernetes
+    branches:
+    - ^master$ # TODO(aramase): enable for release branches
+    labels:
+      # this is required because we want to run kind in docker
+      preset-dind-enabled: "true"
+      # this is required to make CNI installation to succeed for kind
+      preset-kind-volume-mounts: "true"
+      preset-k8s-ssh: "true"
+    annotations:
+      testgrid-dashboards: sig-auth-encryption-at-rest
+      description: Runs conformance tests on a cluster with KMS encryption enabled
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230207-192d5afee3-master
+        # we need privileged mode in order to do docker in docker
+        securityContext:
+          privileged: true
+        command:
+        - runner.sh
+        args:
+        - "/bin/bash"
+        - "-c"
+        - set -o errexit;
+          set -o nounset;
+          set -o pipefail;
+          set -o xtrace;
+          export GO111MODULE=on;
+          go install sigs.k8s.io/kind@v0.17.0;
+          go install sigs.k8s.io/kubetest2@latest;
+          go install sigs.k8s.io/kubetest2/kubetest2-kind@latest;
+          go install sigs.k8s.io/kubetest2/kubetest2-tester-ginkgo@latest;
+          kubetest2 kind -v 5 \;
+            --build \;
+            --up \;
+            --down \;
+            --config test/e2e/testing-manifests/auth/encrypt/kind.yaml \;
+            --cluster-name kms \;
+            --test=ginkgo \;
+            -- \;
+            --focus-regex='\[Conformance\]' \;
```

Comment:
does this regex takes the test that you want?
We avoid serials in presubmits, because they take too long, you should skip them and run with multiple workers, I don't know if kubetest2 already does this by default but we should check

Reply:
it seems kubetets2 has flags to set focus, skip and parallel 

```
	Parallel           int    `desc:"Run this many tests in parallel at once."`
	SkipRegex          string `desc:"Regular expression of jobs to skip."`
	FocusRegex         string `desc:"Regular expression of jobs to focus on."`
```

parallel=2, focus=Conformance, skip=Serial will give you a fast execution with good coverage

Reply:
added `skip=Serial` and `parallel=20` in addition to `focus=Conformance`